frontend/src/components/skeletons/OrganizationDetailsPageSkeleton.tsx

```diff
@@ -0,0 +1,222 @@
+              {Array.from({ length: 2 }, (_, i) => (
+                <div
+                  key={`milestone-${i}`}
+                  className="rounded-lg border-1 border-gray-200 bg-white p-4 shadow-xs dark:border-gray-700 dark:bg-gray-900"
+                >
+                  <Skeleton className="mb-2 h-5 w-full rounded" />
+                  <div className="mt-2 flex flex-wrap items-center text-sm">
+                    <div className="mr-4 flex items-center">
+                      <Skeleton className="mr-2 h-4 w-4 rounded" />
+                      <Skeleton className="h-4 w-24 rounded" />
+                    </div>
+                    <div className="mr-4 flex items-center">
+                      <Skeleton className="mr-2 h-4 w-4 rounded" />
+                      <Skeleton className="h-4 w-20 rounded" />
+                    </div>
+                    <div className="mr-4 flex items-center">
+                      <Skeleton className="mr-2 h-4 w-4 rounded" />
+                      <Skeleton className="h-4 w-16 rounded" />
+                    </div>
+                    <div className="flex flex-1 items-center overflow-hidden">
+                      <Skeleton className="mr-2 h-5 w-4 rounded" />
+                      <Skeleton className="h-4 w-full rounded" />
+                    </div>
+                  </div>
+                </div>
+              ))}
+            </div>
+          </div>
+        </div>
+
+        {/* Two Column Layout - Pull Requests and Releases */}
+        <div className="grid-cols-2 gap-4 lg:grid">
+          {/* Recent Pull Requests */}
+          <div className="mb-8 min-h-[600px] rounded-lg bg-gray-100 p-6 shadow-md dark:bg-gray-800">
+            <h2 className="mb-4 flex flex-row items-center gap-2 text-2xl font-semibold">
+              <Skeleton className="h-5 w-5 rounded" />
+              <Skeleton className="h-6 w-52 rounded" />
+            </h2>
+            <div className="space-y-3">
+              {Array.from({ length: 5 }, (_, i) => (
+                <div
+                  key={`pr-${i}`}
+                  className="rounded-lg border-1 border-gray-200 bg-white p-4 shadow-xs dark:border-gray-700 dark:bg-gray-900"
+                >
+                  <Skeleton className="mb-2 h-5 w-full rounded" />
+                  <div className="mt-2 flex flex-wrap items-center text-sm">
+                    <div className="mr-4 flex items-center">
+                      <Skeleton className="mr-2 h-4 w-4 rounded" />
+                      <Skeleton className="h-4 w-24 rounded" />
+                    </div>
+                    <div className="mr-4 flex flex-1 items-center overflow-hidden">
+                      <Skeleton className="mr-2 h-5 w-4 rounded" />
+                      <Skeleton className="h-4 w-full rounded" />
+                    </div>
+                  </div>
+                </div>
+              ))}
+            </div>
+          </div>
+
+          {/* Recent Releases */}
```

Comment:
The skeletons for Releases doesn't match at all to what we show on the loaded component 🤔 I believe this should be very similar to the rest of the  columns we have:

<img width="1233" height="646" alt="Image" src="https://github.com/user-attachments/assets/1ed706d6-9940-4b91-bb9c-8bf6468f8fa0" />
<img width="1191" height="626" alt="Image" src="https://github.com/user-attachments/assets/5c242a77-a04c-4cef-b4a8-13b6d6098250" />